network-api/networkapi/wagtailpages/factory/__init__.py

```diff
@@ -1,5 +1,4 @@
 from . import (
-    focus_areas,
```

Comment:
this doesn't need a factory, the default areas get built as part of the standard migrations already.